### PR TITLE
Added clear cache

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: Common.Configurations.targets [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.62" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.63" />
     <PackageReference Include="Lt.DigitalOffice.Kernel.EFSupport" Version="1.0.1" />
     <PackageReference Include="LT.DigitalOffice.Kernel.RedisSupport" Version="1.0.4" />
-    <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.6" />
+    <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.7" />
     <PackageReference Include="LT.DigitalOffice.Kernel.ImageSupport" Version="1.0.1" />
     <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.89" />
```
Comment (MariyaSin) [inline Common.Configurations.targets:31]: update to 92

File: src/PositionService.Business/Commands/Position/CreatePositionCommand.cs [hunk 2]
```diff
@@ -22,52 +21,42 @@ public class CreatePositionCommand : ICreatePositionCommand
...
 
-      response.Body = await _repository.CreateAsync(_mapper.Map(request));
-
-      _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
-
-      if (response.Body is null)
+      if (result is null)
       {
-        response = _responseCreator.CreateFailureResponse<Guid?>(HttpStatusCode.BadRequest);
+        return ResponseCreatorStatic.CreateResponse<Guid?>(HttpStatusCode.BadRequest);
```
Comment (MariyaSin) [inline src/PositionService.Business/Commands/Position/CreatePositionCommand.cs:56]: please use ? :

File: src/PositionService.Business/Commands/Position/GetPositionCommand.cs [hunk 3]
```diff
@@ -34,14 +30,10 @@ public async Task<OperationResultResponse<PositionInfo>> ExecuteAsync(Guid posit
...
-        return _responseCreator.CreateFailureResponse<PositionInfo>(HttpStatusCode.NotFound);
+        return ResponseCreatorStatic.CreateResponse<PositionInfo>(HttpStatusCode.NotFound);
       }
 
-      return new()
-      {
-        Status = OperationResultStatusType.FullSuccess,
-        Body = _mapper.Map(position)
-      };
+      return ResponseCreatorStatic.CreateResponse<PositionInfo>(body: _mapper.Map(position));
```
Comment (MariyaSin) [inline src/PositionService.Business/Commands/Position/GetPositionCommand.cs:36]: use ?:

File: src/PositionService.Business/Commands/PositionUser/EditPositionUserCommand.cs [hunk 4]
```diff
@@ -60,46 +46,42 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(EditPositionUserRe
...
+      {
+        await _globalCache.RemoveAsync((await _repository.GetAsync(request.UserId)).PositionId);
       }
 
-      response.Status = response.Body 
-        ? OperationResultStatusType.FullSuccess
-        : OperationResultStatusType.Failed;
-
-      return response;
+      return ResponseCreatorStatic.CreateResponse<bool>(body: result);
```
Comment (MariyaSin) [inline src/PositionService.Business/Commands/PositionUser/EditPositionUserCommand.cs:84]: no need change the command and use ResponseCreator here

Comment (VLiamin) [reply src/PositionService.Business/Commands/PositionUser/EditPositionUserCommand.cs]: done

File: src/PositionService.Business/Commands/Position/GetPositionCommand.cs (hunk 3 above)
Comment (VLiamin) [reply src/PositionService.Business/Commands/Position/GetPositionCommand.cs]: done

File: src/PositionService.Business/Commands/Position/CreatePositionCommand.cs (hunk 2 above)
Comment (VLiamin) [reply src/PositionService.Business/Commands/Position/CreatePositionCommand.cs]: done

File: Common.Configurations.targets (hunk 1 above)
Comment (VLiamin) [reply Common.Configurations.targets:31]: done

**Pull request merged**